File: C/EX07.c
#include <stdio.h>
#include <stdlib.h>

int main (int argc, char const *argv[])
{
    float a, b, c, d, e, S, D, T;
    printf ("Digite o valor do primeiro produto: ");
    scanf ("%f", &a);

    printf ("Digite o valor do segundo produto: ");
    scanf ("%f", &b);

    printf ("Digite o valor do terceiro produto: ");
    scanf ("%f", &c);

    printf ("Digite o valor do quarto produto: ");
    scanf ("%f", &d);

    printf ("Digite o valor do quinto produto: ");
    scanf ("%f", &e);

    S=a+b+c+d+e;

    printf ("A soma dos produtos e igual a %.2f", S);

    printf ("Digite o valor de dinheiro entregue: ");
    scanf ("%f", &D);

    T=D-S;

    printf ("O troco a ser entregue e de %.2f", T);

    return 0;
    
}

File: Python/Ex07.py
a = float(input("Digite o valor do primeiro produto: "))
b = float(input("Digite o valor do segundo produto: "))
c = float(input("Digite o valor do terceiro produto: "))
d = float(input("Digite o valor do quarto produto: "))
e = float(input("Digite o valor do quinto produto: "))

S= a+b+c+d+e

print ("Deve ser pago um total de:", S)

dinheiro = int (input("Digite o valor pago: "))

T= dinheiro - S

print (" Deverá ser devolvido um total de", T, "de troco!")